buildkite/resource_cluster.go

```diff
@@ -75,25 +78,50 @@ func (c *clusterResource) Schema(ctx context.Context, req resource.SchemaRequest
 				MarkdownDescription: "A color representation of the Cluster. Accepts hex codes, eg #BADA55.",
 			},
 		},
+		Blocks: map[string]resource_schema.Block{
+			"timeouts": timeouts.Block(ctx, timeouts.Opts{
```

Comment:
I think we probably want timeouts for all operations right? So using `timeouts.BlockAll()`